test/Core.Test/AdminConsole/OrganizationFeatures/Organizations/OrganizationUpdateCommandTests.cs

```diff
@@ -93,7 +93,7 @@ public async Task UpdateAsync_WhenOrganizationNotFound_ThrowsNotFoundException(
     [Theory]
     [BitAutoData("")]
     [BitAutoData((string)null)]
-    public async Task UpdateAsync_WhenGatewayCustomerIdIsNullOrEmpty_SkipsBillingUpdate(
+    public async Task UpdateAsync_WhenGatewayCustomerIdIsNullOrEmpty_CallsBillingUpdateButHandledGracefully(
```

Comment:
I would remove this test case, it's no longer relevant. It's not blocking through, I can do it in a separate PR if desired.